### DISCUSSION TITLE
Beta Version Kofi - Only one Account log in

### DISCUSSION THREAD

**Discussion by barney1113** [KevinBytesTheDust/CVAmp#182] [General]
I have the problem that he always only makes one account blue with me, with which I was last logged in. He then doesn't want to log in to the others anymore. What could be the reason for the Auth code then expired?

Thank you

**Comment by jlplenio** [comment]
Hey, can you explain the problem in more detail? You are saying that only one of your auth_tokens lead to a successful login?
